### PR TITLE
test: Fix random failures of ThemeLiveReloadIT

### PR DESCRIPTION
**Pull request by mshabarov**
Fixes: https://github.com/vaadin/flow/issues/9746

### CODE REVIEW

**Comment by joheriks** [comment]
Merge commit 87afca55f5e26cb2d950cf6c45ddc1cc48ef3411, which fixes the two failing `AppThemeTestIT` tests.

**Comment by mshabarov** [comment]
For some reason `require('mkdirp')(staticAssetsThemeFolder);` has started to fail when used by copying the theme static resources.
Replaced with `fs.mkdirSync(staticAssetsThemeFolder, {recursive: true});`
Maybe related to a Node.js default version upgrade.

File: flow-tests/test-application-theme/test-theme-live-reload/src/test/java/com/vaadin/flow/uitest/ui/ThemeLiveReloadIT.java [hunk 1]
```diff
@@ -19,98 +19,84 @@
 import java.io.File;
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
-import java.util.UUID;
+import java.util.function.Consumer;
```
Comment (caalador) [inline flow-tests/test-application-theme/test-theme-live-reload/src/test/java/com/vaadin/flow/uitest/ui/ThemeLiveReloadIT.java:22]:
```suggestion
```

**Review by caalador — COMMENTED** [review]
Seems fine, runnable would make it cleaner.
If you take the change use the add suggestion to batch.

**Comment by mshabarov** [comment]
Thanks a lot, @caalador ! Yep, it's much more cleaner now with `Runnable`.

File: flow-tests/test-application-theme/test-theme-live-reload/src/test/java/com/vaadin/flow/uitest/ui/ThemeLiveReloadIT.java [hunk 2]
```diff
@@ -19,98 +19,85 @@
...
             htmlElement.getCssValue("background-color"));
-        createGlobalCssWithBackgroundColor();
+        doActionAndWaitUntilLiveReloadComplete(
+            () -> addBackgroundColorToStylesCSS());
         waitUntilCustomBackgroundColor();
 
         // Live reload upon file deletion
-        deleteFile(globalCSSFile);
+        doActionAndWaitUntilLiveReloadComplete(() ->
+        deleteFile(stylesCSSFile));
```
Comment (caalador) [inline flow-tests/test-application-theme/test-theme-live-reload/src/test/java/com/vaadin/flow/uitest/ui/ThemeLiveReloadIT.java:90]: Minor nit. Formatting is off and this should be indented like on line 100 same for line 84-85 (or the other way around)

**Review by caalador — CHANGES_REQUESTED** [review]

File: flow-tests/test-application-theme/test-theme-live-reload/src/test/java/com/vaadin/flow/uitest/ui/ThemeLiveReloadIT.java (hunk 2 above)
Comment (mshabarov) [reply flow-tests/test-application-theme/test-theme-live-reload/src/test/java/com/vaadin/flow/uitest/ui/ThemeLiveReloadIT.java]: Yep, fixed. Well, now these two lines are fit into one line.

**Review by caalador — APPROVED** [review]

**Pull request merged**